run/custom-metrics/collector/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 Google LLC
+# Copyright 2025 Google LLC
```

Comment:
Please don't update copyright year. This isn't standard practice (just the year it was originally created is fine.

Reply:
Reverted.